### PR TITLE
Remove non-bootstrap code

### PR DESCRIPTION
**Pull request by michaelgugino**
This commit removes steps for nodes that are legacy.

Not using bootstrapping config method is not supported in 3.10.

### CODE REVIEW

**Comment by vrutkovs** [comment]
bot, retest this please

**Comment by vrutkovs** [comment]
GCP flake

/test gcp

File: playbooks/init/evaluate_groups.yml [hunk 1]
```diff
@@ -151,7 +139,6 @@
       ansible_become: "{{ g_sudo | default(omit) }}"
     with_items: "{{ g_new_node_hosts | default(g_node_hosts | default([], true), true) }}"
     changed_when: no
-    when: hostvars[item].openshift_node_bootstrap | default(True) | bool
```
Comment (mtnbikenc) [inline playbooks/init/evaluate_groups.yml]: The group `oo_nodes_to_bootstrap` is now exactly the same as `oo_nodes_to_config`.  We do other things like add masters to this bootstrap group and also set up oo_exclude_bootstrapped_nodes.  This should be cleaned up to simplify the logic.

**Comment by vrutkovs** [comment]
bot, retest this please

File: playbooks/openshift-node/private/additional_config.yml [hunk 2]
```diff
@@ -1,69 +0,0 @@
...
-  # Create group for kuryr nodes
-  - group_by:
-      key: oo_nodes_use_{{ (openshift_use_kuryr | default(False)) | ternary('kuryr','nothing') }}
-    changed_when: False
-
-- import_playbook: etcd_client_config.yml
-  vars:
-    openshift_node_scale_up_group: "oo_nodes_use_flannel:oo_nodes_use_calico:oo_nodes_use_contiv:oo_nodes_use_kuryr"
-
-- name: Additional node config
```
Comment (mtnbikenc) [inline playbooks/openshift-node/private/additional_config.yml]: Just like the kuryr folks found out that their install method needs to change, we should probably let these other groups know these steps are no longer being performed.

**Comment by vrutkovs** [comment]
bot, retest this please

**Comment by mtnbikenc** [comment]
bot, retest this please

**Pull request merged**